contents/handbook/people/time-off.md

```diff
@@ -4,38 +4,39 @@ sidebar: Handbook
 showTitle: true
 ---
 
-At PostHog, we encourage everyone to take time off to recharge, spend time with loved ones, or explore new things.  
-
 We offer our team unlimited time off, but with an expectation that you take _at least 25 days off a year_, including national holidays. This is to make sure that people can take time off flexibly while not feeling guilty about being on vacation.
 
-> We have added some national holidays to our team calendar, but these won't cover everyone. Please decide yourself if you want to take those specific days off. If you do, please remember to book the day off in CharlieHR and hand over any work, especially if you are doing anything customer-facing.  
+The reason for this policy is that it's critical for PostHog that we hire people we can trust to be responsible with their time off - enough that they can recharge, but not so much that it means we don't get any work done. The People & Ops team will look holiday usage occasionally but _only_ to encourage people who haven't taken enough time off to do so. The 25 days is a minimum, not a guide.
```

Comment:
```suggestion
The reason for this policy is that it's critical for PostHog that we hire people we can trust to be responsible with their time off - enough that they can recharge, but not so much that it means we don't get any work done. The People & Ops team will look holiday usage occasionally to encourage people who haven't taken the minimum time off to do so. The 25 days is a minimum, not a guide.
```

I think it's not out of the question that we would look at ie someone who has taken 50 days off as a potential issue if there are no extraneous circumstances (parental leave etc), so I've slightly rephrased